### DISCUSSION TITLE
DAP that works On Windows

### DISCUSSION THREAD

**Discussion by Spotigus** [mrcjkb/rustaceanvim#415] [Q&A]
I for the life of me cannot get a working dap configuration on windows. When running :RustLsp debuggables i alway get : AppData\Local\nvim-data\mason\packages\codelldb/extension/adapte
/codelldb.exe exited with code 101

Runnables works perfectly fine.

Here is a copy of my config.
https://pastebin.com/0zDqjH5g

**Comment by mrcjkb** [comment]
Looks like it's finding codelldb, but there's an issue with your codelldb installation.
I would suggest [searching or asking there](https://github.com/search?q=repo%3Avadimcn%2Fcodelldb+exit+code%3D101&type=discussions) or trying a manual install (not with mason.nvim).
